master/buildbot/test/fake/fakedb.py

```diff
@@ -973,6 +975,7 @@ def fakeAddChangeInstance(self, change):
         row = dict(
             changeid=changeid,
             author=change.who,
+            # committer=change.committer,
```

Comment:
Something is missing here?

Reply:
I deleted that line (if I uncomment it, I get an error something like: `FakeChange object has not attribute committer`)